Setup.bat

```diff
@@ -6,6 +6,15 @@ pushd "%~dp0"
 
 call :MarkStartOfBlock "%~0"
 
+set ProjectDirectory="%1"
+
+if defined ProjectDirectory (
+    echo Project directory for installation is: %ProjectDirectory%
+) else (
+    rem If no argument for the project is provided, assume this script is being run as a plugin within a game project.
+    set ProjectDirectory="%~dp0..\..\.."

```

Comment:
Worth validating that `ProjectDirectory` is valid?

Reply:
Done.